net/usrsock/usrsock_sendto.c

```diff
@@ -376,9 +377,13 @@ ssize_t usrsock_sendto(FAR struct socket *psock, FAR const void *buf,
           goto errout_unlock;
         }
 
+      /* MSG_DONTWAIT is only use in usrsock. */
+
+       flags &= ~MSG_DONTWAIT;
```

Comment:
Should we need handle DONTWAIT like NONBLOCK flag?

Reply:
yes, we need it.